app/src/main/java/com/example/socialapp/network/RetrofitClient.java

```diff
@@ -15,14 +18,16 @@ private RetrofitClient() {
         interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
         OkHttpClient client = new OkHttpClient.Builder().addInterceptor(interceptor).build();
 
-        Retrofit retrofitAuth = new Retrofit.Builder().baseUrl("https://graph.facebook.com/v13.0/")
+        Retrofit retrofitAuth = new Retrofit.Builder()
+                .baseUrl("https://graph.facebook.com/v13.0/")
```

Comment:
Please move this hardcoded url to constant.

Reply:
Comments Implemented.